packages/sdk-btc/source/address.domain.ts

```diff
@@ -0,0 +1,42 @@
+import * as bitcoin from "bitcoinjs-lib";
+
+export const bip44 = (publicKey, network): string =>
```

Comment:
@faustbrian I think I've got the BTC side of things covered. With this 3 functions the generator creates all (legacy, segwit and native segwit addresses). And they produce exactly the same thing my Ledger Nano does. But... read further.